src/monocle_apptrace/utils.py

```diff
@@ -91,3 +92,12 @@ def get_embedding_model() -> str:
     @return: The name of the embedding model, or 'unknown' if not set
     """
     return embedding_model_context.get('embedding_model', 'unknown')
+
+
+def set_context_input(inputs: str, input_key: str):
+    # Set the workflow input in the global context
+    attach(set_value(input_key, inputs))
+
+def get_context_input(input_key: str) -> str:
```

Comment:
setter and getter should be for all attributes.

Reply:
done